config/prohibited.yaml

```diff
@@ -0,0 +1,3 @@
+---
+BANNED_DOMAINS: nsogroup.com,irangov.ir
```

Comment:
Can we use non-existing domain as example ? Something like <prohibiteddomain.com, notagoodfriend.com>

Reply:
@vejmarie sure anything you want ;-)